includes/Modules/AdSense/AMP_Tag.php

```diff
@@ -92,4 +98,47 @@ private function amp_content_add_auto_ads( $content ) {
 		);
 	}
 
+	/**
+	 * Web Story Ad Slot ID.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	private $story_ad_slot_id = '';
+
+	/**
+	 * Set Web Story Ad Slot ID
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $ad_slot_id The Ad Slot ID.
+	 */
+	public function set_story_ad_slot_id( $ad_slot_id ) {
+		$this->story_ad_slot_id = $ad_slot_id;
+	}
+
+	/**
+	 * Adds the AMP Web Story auto ads code if enabled.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function render_story_auto_ads() {
+		printf(
+			'
+				<amp-story-auto-ads>
+					<script type="application/json">
+						{
+							"ad-attributes": {
+							"type": "adsense",
+							"data-ad-client": "%s",
+							"data-ad-slot": "00000000"
+						}
+					}
+				</script>
+				</amp-story-auto-ads>
+				',
+			$this->tag_id, // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			$this->story_ad_slot_id // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		);
+	}
```

Comment:
@benbowler could you please fix indentations here?

Reply:
I noticed there is a stray `}` there too, I've removed that.